scripts/world_bank/health_nutrition_population/population/population.py

```diff
@@ -0,0 +1,117 @@
+import requests
```

Comment:
Pls add a comment block file header similar to:
https://github.com/datacommonsorg/data/blob/master/scripts/world_bank/wdi/worldbank.py

Please add a block comment describing the script as well.